var/spack/repos/builtin/packages/benchmark/package.py

```diff
@@ -44,14 +44,29 @@ class Benchmark(CMakePackage):
         description="The build type to build",
         values=("Debug", "Release", "RelWithDebInfo", "MinSizeRel", "Coverage"),
     )
+    variant(
+        "performance_counters",
+        default=True,
+        description="Enable performance counters provided by libpfm",
```

Comment:
You can make this conditional:
```suggestion
        default=True,
        when="@1.5.4:"
        description="Enable performance counters provided by libpfm",
```

Reply:
@alalazo Thank you for the suggestion. I made this change and removed the conflict.